docs/api/README.md

```diff
@@ -8,18 +8,37 @@ export SPECTER_API_ACTIVE=True
 The Authentication is also necessary if you don't activate any Authentication mechanism.
 In order to make reasonable assumptions about how stable a specific endpoint is, we're versioning them via the URL. Currently, all endpoints are preset with `v1alpha` which pretty much don't give you any guarantee.
 ## Basic Usage
+> Works for the endpoints with `BasicAuthResource`
 
 Curl:
 
 ```bash
-curl -u admin:secret -X GET http://127.0.0.1:25441/api/v1alpha/specter | jq .
+curl -u admin:secret --location --request GET 'http://127.0.0.1:25441/api/v1alpha/token/<jwt_token_id>' | jq .
 ```
 
 Python:
 
 ```python
 import requests
-response = requests.get('http://127.0.0.1:25441/api/v1alpha/specter', auth=('admin', 'secret'))
+response = requests.get('http://127.0.0.1:25441/api/v1alpha/token/<token_id>', auth=('admin', 'secret'))
+json.loads(response.text)
+```
+
+## Token Based Usage
+> Works for the endpoints with `SecureResource` and `AdminResource`
+
+Curl:
+
+```bash
+curl --location --request GET 'http://127.0.0.1:25441/api/v1alpha/specter' \
+--header 'Authorization: Bearer <token>' | jq .
```

Comment:
Please also add the second request which is the one the developer is interested about. Only obtaining the Token adds no Business Value and knowing how to exactly using the token is really critical (and potentially quire time-consuming).

Reply:
> Please also add the second request which is the one the developer is interested about. Only obtaining the Token adds no Business Value and knowing how to exactly using the token is really critical (and potentially quire time-consuming).

@k9ert in this request I'm actually using token to access the `/v1alpha/specter` route.
In the `Basic Usage` section mentioned the user gets the token and in the `Token Usage` section the token is being used.

Isn't this complete:sweat_smile: info?